buddypress/members/single/profile.php

```diff
@@ -247,7 +247,7 @@
                             <path d="M8 9.66602C9.10457 9.66602 10 8.77059 10 7.66602C10 6.56145 9.10457 5.66602 8 5.66602C6.89543 5.66602 6 6.56145 6 7.66602C6 8.77059 6.89543 9.66602 8 9.66602Z" stroke="#737373" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
                         </svg>
                         <?php if($location->location_country === 'OE'): ?>
-                          <?php print __("Online Event", "community-portal"); ?>
+                          <?php _e("Online Event", 'community-portal'); ?>
```

Comment:
Quotes around Online Event need to be changed to single-quote

Reply:
Updated